content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -13,22 +13,11 @@ further_reading:
 ## Compatibility
 
 Supported test frameworks:
-* Jest >=24.8.0
+* Jest >= 24.8.0
   * Only `jsdom` (in package `jest-environment-jsdom`) and `node` (in package `jest-environment-node`) are supported as test environments. Custom environments like `@jest-runner/electron/environment` in `jest-electron-runner` are not supported.
   * Only [`jest-circus`][1] is supported as a `testRunner`.
-* Mocha >=5.2.0
-  * Mocha >=9.0.0 has [partial support](#known-limitations).
-
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+* Mocha >= 5.2.0
+  * Mocha >= 9.0.0 has [partial support](#known-limitations).
```

Comment:
nit: Does this mean mean that mocha 8 is not supported? If mocha 8 is supported, then it seems like different pages use different conventions to talk about versioning. For example, the .NET page states that .NET Core 2.1 and .NET Core 3 are supported.

Reply:
mocha 8 is supported. We support mocha >= 5.2.0 *but* from 9.0.0 on we "lose" _some_ support: we no longer instrument _within_ the test (when using .mjs test files), for example, http requests wouldn't appear in the trace view. 
Makes sense? Do you think there'd be a better way to express this?

Reply:
I'm not sure what the best way to express this is, but it seems the doc isn't consistent between different pages. The other pages define all of the major versions they support whereas this one has the same syntax as the other ones but it ranges over major versions.

Maybe it would be best to adopt a standard syntax like the npm one? That would mean this page is correct, and the other ones would need to either state the minimum major version they support (and support anything above that) or switch to the `^3.0.0` syntax to indicate that they support everything in the 3.x.y range but not necessarily the 4.x.y range. Just an idea.

Reply:
that makes sense. I've created a task to track this as this PR is already huge